Wikipedia/Code/WKWebView+LoadAssetsHtml.m

```diff
@@ -16,11 +16,11 @@ - (void)loadHTMLFromAssetsFile:(NSString*)fileName scrolledToFragment:(NSString*
     
     fragment = fragment ? fragment : @"top";
     NSURL* requestURL = [[WMFProxyServer sharedProxyServer] proxyURLForRelativeFilePath:fileName fragment:fragment];
-    NSURLRequest* request = [NSURLRequest requestWithURL:requestURL];
+    NSURLRequest* request = [NSURLRequest requestWithURL:requestURL cachePolicy:NSURLRequestReloadIgnoringCacheData timeoutInterval:10];
```

Comment:
oh - I can revert this line... I thought WKWebView was caching when I saw "en.wikipedia" as the base href, but really it was being changed by the JavaScript.